dockerfiles/haskell-8.8.Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM haskell:8.8.3-buster
```

Comment:
The haskell-8.8 dockerfile can be removed, we only keep these for backward compatibility purposes (and since grep will start out with haskell-9.0, we don't need backward compatibility)

Reply:
Done.